kvmagent/kvmagent/test/utils/volume_utils.py

```
@@ -107,4 +108,136 @@ def find_volume_controller_by_vol(vm_xml, controller_index):
+# 0000b800  22 00 17 00 00 00 00 00  00 17 00 08 00 00 00 00  |"...............|
+# 0000b810  08 00 7c 09 18 11 1c 3b  20 02 00 00 01 00 00 01  |..|....; .......|
+# 0000b820  01 00 22 00 17 00 00 00  00 00 00 17 00 08 00 00  |..".............|
+# 0000b830  00 00 08 00 7c 09 18 11  1c 3b 20 02 00 00 01 00  |....|....; .....|
+# 0000b840  00 01 01 01 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+# 0000b850  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
+# *
+# 00057000
+def generate_empty_iso_array():
+    a = array.array('B')
+
+    for i in xrange(0x8000):
```

Ruff (0.8.0)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)